tests/queries/test_qs_combinators.py

```diff
@@ -303,7 +303,7 @@ def test_union_with_values_list_on_uncorrelated_models(self):
         gen_y = datetime(1991, 10, 10)
         ReservedName.objects.create(name="Rigel", order=2)
         qs2 = ReservedName.objects.annotate(
-            created=Value(gen_y, DateTimeField())
+            created=Cast(Value(gen_y, DateTimeField()), DateTimeField())
```

Comment:
@charettes I had to add this casting to get the tests to pass on mysql - is this a known mysql quirk or is this something that needs a ticket?

Reply:
I would say it's a known qwirk as the `datetime` literal value is passed as a string through the `MySQLdb` driver. Some backends will always need more help than others passing literal values depending on the type of queries involved.

In other words, I don't think this warrants a ticket.

Thanks for polishing this patch!